simple/src/codingPractice/RemoveDuplicateExample_ReadMe

```diff
@@ -0,0 +1,3 @@
+"C:\Program Files\Java\jdk-17.0.4.1\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\lib\idea_rt.jar=51893:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\bin" -Dfile.encoding=UTF-8 -classpath C:\Users\suyogw\Desktop\Auto\NewMyWayRepo\simple\out\production\simple codingPractice.RemoveDuplicateExample
```

Comment:
RemoveDuplicateExample.java file empty, add code in it

Reply:
Done